Terminal.Gui/Application/Application.cs

```diff
@@ -308,7 +308,7 @@ internal static void ResetState (bool ignoreDisposed = false)
         SupportedCultures = GetSupportedCultures ();
         _mainThreadId = Thread.CurrentThread.ManagedThreadId;
         _initialized = true;
-        InitializedChanged?.Invoke (null, new (false, _initialized));
```

Comment:
Otherwise, if an event is always going to be raised without caring about sender, a delegate that does not include sender should be used, or else it's unclear to subscribers that it will just be ignored.

Reply:
At minimum, using the current Top or something else representing a high level relevant source for the event may also be something to consider, for the call to Invoke.

Reply:
> Not important at the moment, but a general thought:
> 
> Some of these methods should probably have an optional `object? sender = null` parameter, as well, so it can be passed in the event, if specified, since handlers subscribed to it could potentially want to know who originally prompted the event to be raised.

A communications request, @dodexahedron: I really struggle to parse your comments when you use language like "some of these methods". I can't tell what "these" refers to. In this case, the code lines you comment on has two "methods" in them: GetSupportedCultures() and Invoke(). GetSupportedCultures() is completely irrelevant, so I'm pretty sure that's not what you  are referring to. And `Invoke()` is provided by dotnet, so there's no way for me to add an "optional parameter". 

See what I mean?

Reply:
> object? sender = null

`Application.Init/Shutdown` is decoupled from `View`. So `Top` makes no sense. The only object that might make sense is `Application`, but it's a static/singleton so...